pr-deploy/README.md

```diff
@@ -0,0 +1,55 @@
+AMP PR Deploy Bot
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
```

Comment:
`<br>` in Markdown?

Reply:
yep, I didn't want an extra line between the two.